.github/workflows/tests.yml

```diff
@@ -31,6 +31,23 @@ jobs:
       - name: Run make install/uninstall
         run: test/uninstall.sh
 
+  check_size:
+    if: |
+      github.event_name == 'pull_request' &&
+      !contains(github.event.pull_request.labels.*.name, 'bloat-ok')
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        # Use HEAD from the PR instead of the merge commit
+        ref: ${{ github.event.pull_request.head.sha }}
+        # Needed to rebase against the base branch
+        fetch-depth: 0
+    - name: Configure base branch without switching current branch
+      run: git fetch origin ${GITHUB_BASE_REF}:${GITHUB_BASE_REF}
+    - name: Compare binary size change across each commit
+      run: git rebase ${GITHUB_BASE_REF}^ -x ../../test/check-size.sh
```

Comment:
> fatal: cannot run ../../test/check-size.sh: No such file or directory

Reply:
Yes, I mentioned that in the comment above. I tried a few paths, including using the workspace location with `$GITHUB_WORKSPACE/test/check-size.sh`, but the file is not being found for some reason.

Reply:
I think the error makes sense because you are checking out a version which does not have the script yet.

We should be able to ignore this for the initial commit using the mentioned label.

Reply:
Have you tried just `test/check-size.sh`? If this path is not found it is most likely because `git rebase ${GITHUB_BASE_REF}` would checkout to the "main" branch. Since the pull request is not yet merged, the "main" branch does not contain this file.